www_src/components/color-group/color-group.jsx

```diff
@@ -9,9 +9,8 @@ var ColorGroup = React.createClass({
   mixins: [React.addons.LinkedStateMixin],
   getDefaultProps: function () {
     return {
-      colors: this.defaultColors,
-      id: 'value',
-      tinkerUrl: '/users/1/projects/123/pages/0/elements/1/attributes/color/editor/color'
+      colors: this.defaultColors.slice(0),
```

Comment:
no need for `0` thanks to JS magic. If you want a copy `things.slice()` works great.